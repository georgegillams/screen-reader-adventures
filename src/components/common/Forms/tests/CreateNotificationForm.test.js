import React from 'react';
import { render } from '@testing-library/react';

import { CreateNotificationForm } from '..';

describe('<CreateNotificationForm />', () => {
  it('Should render correctly', () => {
    const { container } = render(
      <CreateNotificationForm notification={{ type: 'success' }} onDataChanged={() => null} onSubmit={() => null} />
    );

    expect(container).toMatchSnapshot();
  });
});
