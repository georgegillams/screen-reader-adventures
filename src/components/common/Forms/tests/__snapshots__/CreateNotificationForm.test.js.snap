// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateNotificationForm /> Should render correctly 1`] = `
<div>
  <div
    class=""
  >
    <label
      class="forms__component forms__component__label"
      for="message_FORM_ID"
    >
      Message
    </label>
    <div
      class="input__outer forms__component forms__component__text-box"
      placeholder="Message"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="input__inner"
        id="message_FORM_ID"
        name="Message"
        value=""
      />
      <svg
        class="icon input__icon input__icon--valid"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="forms__component forms__component__label"
      for="type_FORM_ID"
    >
      Type
    </label>
    <div
      class="forms__component forms__component__text-box"
      placeholder="Type"
    >
      <div
        class="input__outer"
      >
        <select
          aria-disabled="false"
          aria-invalid="false"
          class="input__inner input__inner--select"
          id="type_FORM_ID"
          name="Type"
        >
          <option
            value="success"
          >
            Success (green)
          </option>
          <option
            value="warn"
          >
            Warning (orange)
          </option>
          <option
            value="error"
          >
            Error (red)
          </option>
        </select>
        <svg
          class="icon input__icon input__icon--valid"
          height="18"
          viewBox="0 0 24 24"
          width="18"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
          />
        </svg>
      </div>
    </div>
    <button
      class="button__outer button__outer--regular button__outer--no-bouncy forms__component forms__component__button"
      type="button"
    >
      <span
        class="button__child"
      >
        Create notification
      </span>
    </button>
  </div>
</div>
`;
